src/Annotation/ApiResource.php

```diff
@@ -53,7 +53,8 @@
  *     @Attribute("subresourceOperations", type="array"),
  *     @Attribute("sunset", type="string"),
  *     @Attribute("swaggerContext", type="array"),
- *     @Attribute("validationGroups", type="mixed")
+ *     @Attribute("validationGroups", type="mixed"),
+ *     @Attribute("cacheHeaders", type="array")
```

Comment:
that'not what I meant by in alphabetical order :stuck_out_tongue:

Reply:
I'm so dozy, sorry to waste your time like that. I must learn to take time and read comments properly and stop pushing before ready as well. One minute I'll push the change.